packages/dds/sequence/src/sequence.ts

```diff
@@ -260,13 +264,21 @@ export abstract class SharedSegmentSequence<T extends ISegment>
 
 	private messagesSinceMSNChange: ISequencedDocumentMessage[] = [];
 	private readonly intervalCollections: DefaultMap<IntervalCollection<SequenceInterval>>;
+
+	/**
+	 * Wrapper over {@link @fluidframework/shared-object-base#SharedObjectCore.logger}, so we can leverage the extended
+	 * interface internally.
+	 */
+	private readonly loggerExt: ITelemetryLoggerExt;
```

Comment:
This plus setting this new private logger in the DDS' constructor is the basic pattern I used to keep access to the Ext methods, now that `this.logger` inherited from SharedObjectCore is only an `ITelemetryBaseLogger`.